Cargo.toml

```
@@ -4,7 +4,8 @@ version = "0.1.0"
 authors = ["Casper Rogild Storm", "Cory Forsstrom"]
 description = "Halloy is an open-source IRC client written in Rust, with the Iced GUI library. It aims to provide a simple and fast client for Mac, Windows, and Linux platforms."
 documentation = "https://halloy.squidowl.org/"
-license = "LICENSE"
+license = "GPL-3.0-or-later"
+license-file = "LICENSE"
```

```suggestion
license = "GPL-3.0-or-later"
```
`GPL-3.0-or-later` is in SPDX, so `lincese-file` should *not* be included. The file is already packaged as part of the crate.

---

Hmm, ok. It's because https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields says "If a package is using a nonstandard license, then the license-file field may be specified in lieu of the license field." Understood.